### PR TITLE
Fixing CYA for Transport

### PR DESCRIPTION
**Pull request by versatil56**

### CODE REVIEW

File: app/pages/transport/AddAnySealsPage.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ case object AddAnySealsPage extends QuestionPage[Boolean] {
   override def navigateInNormalMode(answers: UserAnswers): Call =
     answers.get(AddAnySealsPage) match {
       case Some(true) => transportRoutes.SealController.onPageLoad(NormalMode, answers.lrn, Index(0))
-      case Some(false) => routes.CheckYourAnswersController.onPageLoad(answers.lrn)
-      case None => routes.JourneyRecoveryController.onPageLoad()
+      case Some(false) => transportRoutes.CheckTransportController.onPageLoad(answers.lrn)
+      case None => transportRoutes.CheckTransportController.onPageLoad(answers.lrn)
```
Comment (AdamCoops) [inline app/pages/transport/AddAnySealsPage.scala:36]: Are we deliberately not going to JourneyRecovery here?

Comment (giftig) [inline app/pages/transport/AddAnySealsPage.scala:36]: Shouldn't this one remain as journey recovery? From what I understand we should never be hitting this navigate if the question wasn't answered.

File: app/pages/transport/AddSealPage.scala [hunk 2]
```diff
@@ -38,8 +38,8 @@ case object AddSealPage extends QuestionPage[Boolean] {
       }
     } else {
       mode match {
-        case NormalMode => routes.CheckYourAnswersController.onPageLoad(answers.lrn)
-        case CheckMode => routes.CheckYourAnswersController.onPageLoad(answers.lrn)
+        case NormalMode => transportRoutes.CheckTransportController.onPageLoad(answers.lrn)
+        case CheckMode => transportRoutes.CheckTransportController.onPageLoad(answers.lrn)
```
Comment (giftig) [inline app/pages/transport/AddSealPage.scala:42]: It's going to the same place in any case so I guess we can simplify this by just removing the whole match while we're here.

File: app/viewmodels/checkAnswers/transport/OverallDocumentSummary.scala [hunk 3]
```diff
@@ -36,5 +41,22 @@ object OverallDocumentSummary {
             .onPageLoad(NormalMode, answers.lrn)
             .url
         )
-    }.toList
+    }
+
+  def checkAnswersRow(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
+    answers.get(AllOverallDocumentsQuery).map {
+      documents =>
+        val value = documents.map(c=>s"${c.documentType.name}(${c.reference})").mkString("<br>")
```
Comment (giftig) [inline app/viewmodels/checkAnswers/transport/OverallDocumentSummary.scala:49]: can you add spaces around the `=>` please?

File: app/pages/transport/AddAnySealsPage.scala (hunk 1 above)
Comment (versatil56) [reply app/pages/transport/AddAnySealsPage.scala:36]: no - we are not. This is a mistake! Fixing now!

File: app/viewmodels/checkAnswers/transport/SealSummary.scala [hunk 4]
```diff
@@ -36,5 +42,22 @@ object SealSummary  {
             .onPageLoad(NormalMode, answers.lrn)
             .url
         )
-    }.toList
+    }
+
+  def checkAnswersRow(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
+    answers.get(AllSealsQuery).map {
+      seals =>
+        val value = seals.map(c=>c).mkString("<br>")
```
Comment (giftig) [inline app/viewmodels/checkAnswers/transport/SealSummary.scala:50]: You can ditch the identity map

File: app/views/transport/CheckTransportView.scala.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.routes
+
+@this(
+        layout: templates.Layout,
+        govukSummaryList: GovukSummaryList
```
Comment (giftig) [inline app/views/transport/CheckTransportView.scala.html:21]: weird indentation there

**Review by giftig — CHANGES_REQUESTED** [review]

**Review by AdamCoops — DISMISSED** [review]

**Review by giftig — APPROVED** [review]

**Pull request merged**